pkg/deploy/internal/document/document.go

```diff
@@ -38,16 +38,16 @@ import (
 type Client interface {
```

Comment:
Does this interface need to be public?
Either make it private or (in the line above) create it in `document_mock.go` (without `_test` suffix

Reply:
Yes, it does as the type is used in the arguments of the `Deploy(...)` function, which itself is used from outside the package.